src/main/java/net/dv8tion/discord/Yui.java

```diff
@@ -201,11 +200,6 @@ private static void setupBot()
             System.out.println("Our login thread was interrupted!");
             System.exit(UNABLE_TO_CONNECT_TO_DISCORD);
         }
-        catch (RateLimitedException e)
-        {
-            System.out.println("Encountered ratelimit while attempting to login!");
-            System.exit(UNABLE_TO_CONNECT_TO_DISCORD);
-        }
```

Comment:
Why was this removed?

Reply:
Because as of JDA 3.5 this is no longer thrown. see changelog <https://gist.github.com/MinnDevelopment/db2b5add3a8e4a4c8c1d72da778edd57#changes>